cocos2d/core/value-types/color.ts

```diff
@@ -185,7 +185,7 @@ export default class Color extends ValueType {
      * Set the components of a color to the given values.
      * @method set
      * @typescript
-     * set (out: Color, r = 255, g = 255, b = 255, a = 255): Color
+     * set (out: Color, r?: number, g?: number, b?: number, a?: number): Color
```

Comment:
Why to remove the default param value

Reply:
When we compile typescript with `tsc`command, we will get this error :

> error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

Reply:
Microsoft is also discussing a better way to express default values ( https://github.com/Microsoft/tsdoc/issues/151 ),  but now it's not the right way:
`set (out: Color, r = 255, g = 255, b = 255, a = 255): Color
`

Reply:
I see, thanks.